src/js/components/RequestErrorMsg.js

```diff
@@ -9,14 +9,16 @@ import Config from "../config/Config";
 
 function getDefaultMessage() {
   return (
-    <Trans render="p" className="text-align-center flush-bottom">
-      You can also join us on our{" "}
-      <a href={Config.slackChannel} target="_blank">
-        Slack channel
-      </a>{" "}
-      or send us an email at{" "}
+    <p className="text-align-center flush-bottom">
+      <Trans>
+        You can also join us on our{" "}
+        <a href={Config.slackChannel} target="_blank">
+          Slack channel
+        </a>{" "}
+        or send us an email at
+      </Trans>{" "}
       <a href={`mailto:${Config.supportEmail}`}>{Config.supportEmail}</a>.
```

Comment:
let's move the email into the `Trans`, so it can be moved within the sentence for other languages.

Reply:
Unfortunately, being inside <Trans> is what breaks it and I couldn't figure out how to keep it there while also showing it correctly.

Reply:
now that makes sense. sorry for not paying close attention to the diff.

the perfect opportunity for me to look at lingui. :)